### ISSUE TITLE
Appender default flushlevel is inconsistent with java-logging client's default

### ISSUE THREAD

**Issue by burkedavison** [googleapis/java-logging-logback#1328]
The java-logging library [updated its flushlevel default](https://github.com/googleapis/google-cloud-java/pull/4254) to `OFF` in Dec 2018 due to the performance cost of flushing on the same thread as the application.

However, the java-logging-logback appender was [implemented in May 2017 to default to `ERROR`](https://github.com/googleapis/java-logging-logback/commit/5a11507f0fba93ae788863bc4776444d46b2c112), and continues to default to `ERROR`.

( Add'l context: This came to our attention with a request to make Spring Cloud GCP's helper-implementation of a logback appender match the base library's default. See https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/2906 for discussion. )

**Comment by cindy-peng** [comment]
Good catch! Thanks for reporting @burkedavison! We will work on a fix here.
